lookout/style/typos/tests/test_analyzer.py

```diff
@@ -45,30 +46,27 @@ def setUpClass(cls):
                                    language="Python")]
         cls.ptr = ReferencePointer("someurl", "someref", "somecommit")
 
-    @unittest.skip
     def test_train(self):
         datastub = FakeDataStub(files=self.base_files, changes=None)
         model = IdTyposAnalyzer.train(self.ptr, {}, datastub)
         self.assertIsInstance(model, DummyAnalyzerModel)
 
-    @unittest.skip
     def test_analyze(self):
         datastub = FakeDataStub(files=self.base_files,
                                 changes=[Change(base=self.base_files[0], head=self.head_files[0])])
         model = IdTyposAnalyzer.train(self.ptr, {}, datastub)
-        analyzer = IdTyposAnalyzer(model, self.ptr.url, {})
+        analyzer = IdTyposAnalyzer(model, self.ptr.url, config=dict(
+                model=MODEL_PATH, confidence_threshold=0.0, n_candidates=3))
```

Comment:
why do you have different `confidence_threshold=0.2` & `confidence_threshold=0.0` in different tests?
Not only the number of comments but comments itself should be checked here too - that they have everything as expected.
Model is fixed, so results should be deterministic.

Reply:
Test with 0.2 has fixed output check, while the one with 0.0 aims at getting any comments at all.
For now I am not aware of the comments structure, so it would be better if you check and add this determinism it here. To me it looks like here we can only check comments format: the test corrector is very very dumb (by design, to be very small), so I don't have any expectations of reasonable corrections.